source/includes/administration/_notification_categories.md

```diff
@@ -0,0 +1,235 @@
+
+
+<!-------------------- GET NOTIFICATION CATEGORY -------------------->
+
+#### Retrieve notification category
+
+`GET /content/categories/:id`
+
+```shell
+# Retrieve notification category
+curl "https://cloudmc_endpoint/v2/rest/content/categories/:id?language=:language" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns JSON structured like this:
+
+```json
+{
+    "data": {
+        "organizationId": "3e824205-f507-4ffe-9891-a8addf3cf0e9",
+        "createdAt": "2020-10-29T14:06:50.000Z",
+        "translations": [
+            {
+                "language": "en",
+                "id": "1cea5fe9-3bfa-4eea-a9aa-8e8aa8f85346",
+                "text": "Test-notif-reseller",
+                "type": "title",
+                "version": 1
+            },
+            {
+                "language": "fr",
+                "id": "d4044366-2bf9-4c1f-9da5-e2e47b719714",
+                "text": "test-notif-reseller",
+                "type": "title",
+                "version": 1
+            }
+        ],
+        "icon": "fa fa-envelope",
+        "id": "dc921a2a-16c2-4bc3-8ed6-bcb571bae241",
+        "version": 1,
+        "updatedAt": "2020-10-29T14:06:50.000Z"
+    }
+}
+```
+Get a specific notification category.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the notification category.
+`organizationId`<br/>*UUID* | The organization id of the notification category.
+`createdAt`<br/>*string* | The date on which the notification category was created.
+`updatedAt`<br/>*string* | The date on which the notification category was updated.
+`icon`<br/>*string* | The icon used when displaying the notification category.
+`version`<br/>*integer* | The notification category's version.
+`translations`<br/>*Array[Object]* | The translations of the notification category's content.
+`translations.language`<br/>*string* | The language of that the translation is in.
+`translations.text`<br/>*string* | The content of the notification.
+`translations.type`<br/>*string* | The content that we are translating. (Must be one of: title, body, description, url slug)
+`translations.id`<br/>*UUID* | The translation's id.
+`translations.version`<br/>*integer* | The translation's version
+
+<!-------------------- CREATE NOTIFICATION CATEGORY -------------------->
+
+#### Create notification category
+
+`POST /content/categories/`
+
+```shell
+# Create notification category
+curl -X POST "https://cloudmc_endpoint/v2/rest/content/categories/" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json" \
+   -d "request-body"
+```
+> Request body example:
+
+```json
+{
+    "organizationId":"2cc1b5ba-2cfb-4bf9-9cb9-c67d34ba27b1",
+    "icon": "fa fa-envelope",
+    "translations": [
+        {
+            "language":"en",
+            "type":"title",
+            "text":"trial-ttt"
+        },
+        {
+            "language":"fr",
+            "type":"title",
```

Comment:
is only the title required?

Reply:
Yes, there is no body for the notification categories as far as I can tell.

Reply:
This is information that you should specify in the description at line 166

Reply:
Added this in the field description for translation.type 65d3185551293faa443a5e59617d81963fb934f1